src/__tests__/extensions/surveys.test.ts

```diff
@@ -1043,67 +1043,6 @@ describe('useToggleSurveyOnURLChange', () => {
         expect(mockSetSurveyVisible).not.toHaveBeenCalled()
     })
 
-    it('should show survey when URL changes to match conditions', () => {
```

Comment:
if there was a test for it, I wonder if there was a reason for it and now we are changing it?

Reply:
Is there an issue caused by this code path so I can get better context?

Reply:
this was a test i added on my [last PR](https://github.com/PostHog/posthog-js/pull/1755), however, i was doing some tests locally, and this is causing delayed surveys to overcome the delay and also skip the `survey sent` event, since thats handled by the `showSurvey()` method